pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-libnotify-plus/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, pidgin, glib, libnotify, purple-events, autoreconfHook, pkgconfig, intltool }:
+
+stdenv.mkDerivation rec {
+  pname = "purple-libnotify-plus";
+  version = "2018-05-30";
```

Comment:
Same here: Why not use https://github.com/sardemff7/purple-libnotify-plus/releases/tag/v2.99.1 ? It's only 7 commits behind `master`: https://github.com/sardemff7/purple-libnotify-plus/compare/v2.99.1...master